integreat_cms/cms/views/pois/poi_actions.py

```diff
@@ -13,8 +13,6 @@
 
 from linkcheck.models import Link
 
-from geopy.point import Point
```

Comment:
I think the view should not need to import any third-party code to use public functions of the nominatim API client - instead, we should perform the conversion from float numbers to a `Point` object in the client itself.